--- /usr/share/phoronix-test-suite/pts-core/commands/result_file_to_pdf.php	2016-03-28 18:45:40.000000000 +0000
+++ /usr/share/phoronix-test-suite/pts-core/commands/result_file_to_pdf.php.new	2020-06-10 21:58:51.618303543 +0000
@@ -124,7 +124,7 @@
 		*/
 		$pdf_file = pts_client::user_home_directory() . $r[0] . '.pdf';
 
-		$pdf->Output($pdf_file);
+		$pdf->Output($pdf_file, 'F');
 		echo PHP_EOL . 'Saved To: ' . $pdf_file . PHP_EOL;
 	}
 	public static function invalid_command($passed_args = null)
